### ISSUE TITLE
Intuition behind sliding window analysis

### ISSUE THREAD

**Issue by crmclean** [crmclean/Autotuner#25]
Thanks for the quick response. That fixed the problem.
Separately, I also wonder whether you can shed some light on how you choose the lag, threshold and influence parameters? Is it just trial and error for the particular data using different combinations or is there a method to it using data from the scans?

_Originally posted by @BaylorSci in https://github.com/crmclean/Autotuner/issues/24#issuecomment-625890663_

**Comment by Phylloxera** [comment]
Looks like this was left open to prompt discussion? If so, good since it is not really a technical issue but maybe more of an artistic/visual issue? I'm attaching my plots of the tutorial settings on the tutorial data and then one where I lowered the lag to 20. The second definitely has more peaks (45 vs 39 by my count)... The question is, does it have more 'narrow' peaks? Tough for my eyeballs on my 11X7 macbook air screen to tell :)
![lag25](https://user-images.githubusercontent.com/13121075/100122820-75da7300-2e3f-11eb-8fc9-27439bec5c04.png)
![lag20](https://user-images.githubusercontent.com/13121075/100122841-7e32ae00-2e3f-11eb-9706-ec5df73969c7.png)

**Comment by crmclean** [comment]
Hi @Phylloxera,

Cool name! This was left issue unresponded because I kind of dropped the ball here and forgot. Thanks for reminding me to address it. I have been the general question about the sliding window a few times before via email, and this is my usual response:

The choice of lag, threshold, and influence should have zero effect on the parameter estimation processes, assuming the same set of TIC peaks are used for the parameter estimation steps of AT. If a common peak is selected between two sets of signal processing parameters, and the width of the peak varies, AT uses an internal function to expand the peak and consider any EIC that may have data beyond the predetermined bounds. The key thing here is that you are making estimates from the most intense TIC peaks, as I would imagine that those peaks are not coming from increases in the baseline or noise. So my recommendation to you if your objective is to find optimal signal processing parameters would be to review the output of the function plot_peaks. If the peaks look like legitimate peaks, then I would feel confident that you are performing parameter estimation through statistical inference of the raw data of actual features. If you'd like to be even more careful, I would recommend increasing the number of peaks analyzed by AT using the parameter 'returned_peaks' from the function 'isolatePeaks'. Assuming all peaks are real, the more peaks you analyze, the closer the estimates should be to some kind of normal distribution since the algorithm can consider around 10k observations per peak depending on the mass analyzer you are using the collect your data. 

Now, I would like to address your specific question, @Phylloxera. It's hard for me to tell whether your imagines do or do not have more or less narrow peaks. The width should be corrected for in the "EICparams" function though.

**Comment by Phylloxera** [comment]
OK, thank you. That will maybe make more sense to me once I get to those later sections of the tutorial. I know I'll have more questions regarding my own (Waters MSE) data later. Is email better for general questions?

**Comment by crmclean** [comment]
Yup. Email is the way to go. Please write to me at crmclean@mit.edu
